.github/workflows/gitflow-ci.yaml

```
@@ -1,132 +1,58 @@
 
-      - name: Setup Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: ${{ env.NODE_VERSION }}
-          cache: 'npm'
+      # Login to Docker Hub
 
-      - name: Install dependencies
-        run: npm ci
+      - name: Docker login
+        uses: docker/login-action@v2
```

actionlint (1.7.7)

the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)